job-simulation.rb

```diff
@@ -1,4 +1,62 @@
 require './Stack.rb'
 require './Queue.rb'
 
+class CrappyJob
+  attr_reader :waiting, :workers
+
+  def initialize
+    @waiting = Queue.new
+    @workers = Stack.new
+  end
+
+  # method to add people to the waiting list queue
+  def wait_list(num)
+    # num.downto(1).each do |n|
+    (1..num).each do |n|
+      @waiting.enqueue(n)
+    end
+    # test
+    # puts "First wait list #{waiting.store}"
+  end
```

Comment:
I like this, but I think that you could (should?) have put this in the initialize method. Basically something like this http://ruby-doc.org/docs/ruby-doc-bundle/UsersGuide/rg/objinitialization.html where by default you add 10 people to wait from the get-go, otherwise the number that the user gives.